test/testParametric.jl

```diff
@@ -28,21 +29,28 @@ for i in 0:3
     addFactor!(fg, [psym;nsym], pp)
 end
 initAll!(fg)
-
 IIF.initParametricFrom!(fg)
 
+vars = ls(fg, r"x") 
+lands = ls(fg, r"l") 
+results = IIF.autoinitParametric!.(fg, [vars;lands])
+
+# pl = plotSLAM2D(fg, solveKey=:parametric, drawContour=false, xmin=-20, xmax=20, ymin=-20, ymax=20)
+
+
 vardict, result, varIds, Σ = IIF.solveGraphParametric(fg) #autodiff=:finite)
 
 #TODO test +-pi used pi+1e-5
 #FIXME look if something is wrong with angle bounds [-pi,pi), test failed
+M = getManifold(Pose2)
 # @test isapprox(vardict[:x0].val, [10, 10, -pi], atol = 1e-3)
 # @test isapprox(vardict[:x4].val, [10, 10, -pi], atol = 1e-3)
-
-@test isapprox(vardict[:x0].val, [10, 10, -pi], atol = 1e-3) || isapprox(vardict[:x0].val, [10, 10, pi], atol = 1e-3)
-@test isapprox(vardict[:x1].val, [0, 10, -pi/2], atol = 1e-3)
-@test isapprox(vardict[:x2].val, [0, 0, 0], atol = 1e-3)
-@test isapprox(vardict[:x3].val, [10, 0, pi/2], atol = 1e-3)
-@test isapprox(vardict[:x4].val, [10, 10, -pi], atol = 1e-3) || isapprox(vardict[:x4].val, [10, 10, pi], atol = 1e-3)
+ϵ = getPointIdentity(M)
+@test isapprox(M, vardict[:x0].val, exp(M, ϵ, hat(M,ϵ,[10, 10, -pi])), atol = 1e-3)
```

Comment:
```suggestion
@test isapprox(M, vardict[:x0].val, exp_lie(M, hat(M,ϵ,[10, 10, -pi])), atol = 1e-3)
```

Reply:
Ronny suggested this on SO(3) stuff I was doing somewhere

Reply:
It defaults to ProductRepr and will give an error. Plus its the wrong exp.

Reply:
ah, okay